src/components/Adyen/Adyen.js

```diff
@@ -288,17 +289,21 @@ const Adyen = ({
     const isBancontactCard =
       selectedPaymentMethodRef?.current?.methodName === 'bancontact_card';
 
-    let checkbox = document.querySelector(`.checkbox-${methodName}`);
+    let checkbox = document.querySelector(`#checkbox-${methodName}`);
```

Comment:
i think that having an ID and a CSS class with the same names, but pointing to different elements is confusing. I suggest doing something more specific, so the names kinda explain themselves
like, `id='${methodName}-input`
and the class for the label (i assume the `checkboxWrapper` points to a `<label>`) would have either a CSS class or an ID with a value like `${methodName}-inputLabel`

Reply:
Will change this wrapper (div element) to label and those selectors names